statics/css/skydive.css

```diff
@@ -508,20 +508,29 @@ circle.manager {
   margin-top: 0px;
 }
 
-.dynamic-table .fa-cog, .dynamic-table .btn, .dynamic-table .btn[active], .dynamic-table .btn[focus] {
+.dynamic-table .fa-cog, .dynamic-table .btn, .dynamic-table .btn[active], .dynamic-table .btn[focus],
+.object-detail-filter .fa-cog, .object-detail-filter .btn, .object-detail-filter .btn[active],
```

Comment:
you could alternatively leave this block as is and then append the `dynamic-table` class as a secondary class such in:

```
 <div v-if="filterIcon" class="object-detail-filter">\
 ...
```

Would change to:

```
 <div v-if="filterIcon" class="object-detail-filter dynamic-table">\
  ...
```

Reply:
done